metricflow/test/generate_snapshots.py

```diff
@@ -73,7 +73,7 @@ class MetricFlowTestCredentialSetForAllEngines(FrozenBaseModel):  # noqa: D
     big_query: MetricFlowTestCredentialSet
     databricks: MetricFlowTestCredentialSet
     postgres: MetricFlowTestCredentialSet
-    trino: MetricFlowTestCredentialSet
+    # trino: MetricFlowTestCredentialSet
```

Comment:
Please revert this and regenerate the Trino things. I think it should all be working now.

Reply:
Oops, don't think I meant to commit that one!